### PR TITLE
create-lambda-for-putting-comments

### PR DESCRIPTION
**Pull request by nicolascr181**
image
image

### CODE REVIEW

File: dashboard/src/components/ShowEvent.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@
               ></v-rating>
 
               <div class="grey--text ms-4">
-                4.5 (413)
+                {{this.model.score}} ({{this.model.total_comments}})
```
Comment (1701722767) [inline dashboard/src/components/ShowEvent.vue:35]: why duplicate the score?

File: services/events/functions/create/main.py [hunk 2]
```diff
@@ -56,7 +55,9 @@ def create_event(event):
         "slots": event['slots'],
         "max": event['max'],
         "datestart": event['datestart'],
```
Comment (1701722767) [inline services/events/functions/create/main.py]: I suggest date_start is a better nomenclature

**Review by 1701722767 — CHANGES_REQUESTED** [review]

File: dashboard/src/components/ShowEvent.vue (hunk 1 above)
Comment (nicolascr181) [reply dashboard/src/components/ShowEvent.vue:35]: One of them is the attr of the component, the stars , the other one is for the user to see. That's how it was made the v-card component.

Comment (nicolascr181) [reply dashboard/src/components/ShowEvent.vue:35]: One of them is the attr of the component, the starts , the other one is for the user to see. That's how it was made the v-card component.

Comment (nicolascr181) [reply dashboard/src/components/ShowEvent.vue:35]: image

**Comment by nicolascr181** [comment]
image

**Review by 1701722767 — APPROVED** [review]

**Review by sergiomonz — APPROVED** [review]

**Pull request merged**